[BE] Clean up

MihaiDruta opened the pull request:
[BE] Clean up
https://trello.com/c/8it1dbRd/56-be-clean-up

Code clean up. Fixing minor issues. CreatedAt/UpdatedAt automatization.

Commit:
[BE] Clean up
https://trello.com/c/8it1dbRd/56-be-clean-up

Code clean up. Fixing minor issues. CreatedAt/UpdatedAt automatization.

SpartakusMd commented on src/Service/ActivityTransformer.php:
```diff
@@ -110,39 +103,68 @@ private function resetTechTypeCollections(Activity $entity): void
+            );
+            throw $entityNotFound;
+        }
+        $activity->setOwner($tempUser);
```
The owner can't be changed. It is set once at creation and that's all.

SpartakusMd commented on src/Service/ActivityTransformer.php:
```diff
@@ -110,39 +103,68 @@ private function resetTechTypeCollections(Activity $entity): void
         $entity->setStatus($dto->status);
 
         /** @var User $tempUser */
         $tempUser = $this->userRepo->find($dto->owner);
```
The current user should be passed from the controller as the owner. With this change I could create a job with someone else as owner.

Commit:
[BE] Clean up
https://trello.com/c/8it1dbRd/56-be-clean-up

Code clean up. Fixing minor issues. CreatedAt/UpdatedAt automatization.

Commit:
[BE] Clean up
https://trello.com/c/8it1dbRd/56-be-clean-up

Code clean up. Fixing minor issues. CreatedAt/UpdatedAt automatization.
